app/src/main/res/menu/activity_main_drawer.xml

```diff
@@ -17,9 +17,9 @@
             android:icon="@drawable/ic_menu_slideshow"
             android:title="Shelter Profile" />
         <item
-            android:id="@+id/nav_settings"
+            android:id="@+id/nav_user_capabilities"
```

Comment:
in the mainactivity, perhaps in the setUpLoginView or whatever I called it, or after login, only show this when user is logged in (hidden by default). Then hide it again when user logs out. 

Similar logic to the login/logout buttons